tests/test_mdlp.py

```diff
@@ -32,165 +34,52 @@ def test_find_cut_no_cut():
     k = find_cut(y, 0, len(y))
     assert_equal(-1, k)
 
-def test_mdlp_iris():
-    from sklearn.datasets import load_iris
+def test_e2e():
+  X = np.array([
+    [0.1, 0.1],
```

Comment:
So instead of having decimal values here, why not multiply each value by 10? Discretization should be invariant under scaling.

Reply:
I think that having decimals for the initial test data is reasonable as a way to emphasize that discretization operates on continuous data. The invariance under scaling is interesting though. I will add several assertions demonstrating the invariance of discretization under several transformations of the data.